traffic_monitor/cache/cache_test.go

```diff
@@ -95,3 +100,37 @@ func TestComputeStatGbps(t *testing.T) {
 		}
 	}
 }
+
+func TestParseAndDecode(t *testing.T) {
+	file, err := ioutil.ReadFile("stats_over_http.json")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pl := &poller.HTTPPollCtx{HTTPHeader: http.Header{}}
+	ctx := interface{}(pl)
+	ctx.(*poller.HTTPPollCtx).HTTPHeader.Set("Content-Type", "text/json")
```

Comment:
`text/json` is not an IANA-standard MIME-type. I think you mean `application/json`.

Which is a nice segue into mentioning that these should be using the relevant constants from the [`github.com/apache/trafficcontrol/lib/go-rfc` package](https://pkg.go.dev/github.com/apache/trafficcontrol/lib/go-rfc#pkg-constants)